src/config/constants/common.ts

```diff
@@ -36,3 +36,10 @@ export const POSITIONS_VERSIONS: PositionVersions[] = [
 export const TOKEN_TYPE_BASE = 'BASE';
 export const TOKEN_TYPE_WRAPPED = 'WRAPPED_PROTOCOL_TOKEN';
 export const TOKEN_TYPE_YIELD_BEARING_SHARES = 'YIELD_BEARING_SHARES';
+
+export const TOKEN_BLACKLIST = [
+  '0x5fe2b58c013d7601147dcdd68c143a77499f5531', // POLY - GRT
```

Comment:
We can create a multisig tx to disable this token, so users won't be able to deposit or add funds anyway. So we should be able to remove it from the list